app/mailers/user_notifier_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class UserNotifierMailer < ApplicationMailer
+
+
+	default :from => 'company@sparkstart.org'
+
+	# send a signup email to the user, pass in the user object that   contains the user's email address
+	def send_signup_email(user)
+	  @user = user
+	  mail( :to => @user.email,
+	  :subject => 'Thanks for signing up for Spark Start!' )
```

Comment:
Can the :subject symbol be easily changed going forward? Without getting into the back-end?